src/containers/shared/components/Transaction/XChainAddAttestation/AccountCreateAttestation.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { Amount } from '../../Amount'
+import { SimpleRow } from '../SimpleRow'
+
+export interface AccountCreateAttestationProps {
+  send: string
+  account: string
+  destination: string
+  signature: string
+}
+
+export const AccountCreateAttestation = (
+  props: AccountCreateAttestationProps,
+) => {
+  const { t } = useTranslation()
+  const { send, account, destination, signature } = props
+
+  return (
+    <div className="claim-attestation" key={signature}>
+      <div className="claim-attestation-title">
+        XChain Create Account Attestation Batch Element
+      </div>
+      <SimpleRow label={t('send')} data-test="send">
+        <Amount value={send} />
+      </SimpleRow>
+      <SimpleRow label={t('account')} data-test="account">
+        <Account account={account} link={false} />
```

Comment:
Why is one of these links deactivated? (Is it because the sending account is not on this network? - if so, is it possible to link to the account on the correct network?)

Reply:
The account is not on the network. You can't link to the account on the correct network because this network has no clue what the other network is.